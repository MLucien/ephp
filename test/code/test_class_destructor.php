<?php

class Simple {

    public function __construct() {
        print "Constructing...\n";
    }

    public function __destruct() {
        print "Destructing...\n";
    }
}

$s1 = new Simple();
$s2 = new Simple();
$s3 = array( new Simple(), 1, 2, 3 );
unset($s1);
unset($s3);
