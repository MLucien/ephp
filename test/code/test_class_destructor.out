Constructing...
Constructing...
Constructing...
Destructing...
Destructing...
Destructing...
